[debug] [2018-02-27T09:50:05.857Z] ----------------------------------------------------------------------
[debug] [2018-02-27T09:50:05.857Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\aAkIb\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login
[debug] [2018-02-27T09:50:05.857Z] CLI Version:   3.17.4
[debug] [2018-02-27T09:50:05.857Z] Platform:      win32
[debug] [2018-02-27T09:50:05.857Z] Node Version:  v9.4.0
[debug] [2018-02-27T09:50:05.857Z] Time:          Tue Feb 27 2018 15:20:05 GMT+0530 (India Standard Time)
[debug] [2018-02-27T09:50:05.857Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=930603619&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2018-02-27T09:50:31.874Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token  
 { code: '4/AABVvIChir1iYdX4eDBYrwbnIGwyzA40BYI-BiJJb_IO43BLhR0qd3_4DbMui1OeN_I1_ALQxhE20F8y3X0KijI',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Tue Feb 27 2018 15:20:31 GMT+0530 (India Standard Time)
[debug] [2018-02-27T09:50:33.159Z] Token Fetch Error: Error
    at new FirebaseError (C:\Users\aAkIb\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:17)
    at Request._callback (C:\Users\aAkIb\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:34:23)
    at self.callback (C:\Users\aAkIb\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:186:22)
    at Request.emit (events.js:160:13)
    at Request.onRequestError (C:\Users\aAkIb\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:878:8)
    at ClientRequest.emit (events.js:160:13)
    at TLSSocket.socketErrorListener (_http_client.js:389:9)
    at TLSSocket.emit (events.js:160:13)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at process._tickCallback (internal/process/next_tick.js:152:19)
